doc/cephadm/services/rgw.rst

```diff
@@ -83,16 +83,16 @@ To deploy RGWs serving the multisite *myorg* realm and the *us-east-1* zone on
 
 .. prompt:: bash #
 
-   ceph orch apply rgw east --realm=myorg --zone=us-east-1 --placement="2 myhost1 myhost2"
+   ceph orch apply rgw east --realm=myorg --zonegroup=us-east-zg-1 --zone=us-east-1 --placement="2 myhost1 myhost2"
 
 Note that in a multisite situation, cephadm only deploys the daemons.  It does not create
-or update the realm or zone configurations.  To create a new realm and zone, you need to do
-something like:
+or update the realm or zone configurations.  To create a new realms, zones and zonegroups
+you can use :ref:`mgr-rgw-module` or manually using something like:
 
 .. prompt:: bash #
 
   radosgw-admin realm create --rgw-realm=<realm-name> --default
```

Comment:
I don't think you need `--default` for the below realm/zonegroup/zone create commands below. That flag only applies if the realm/zg/zone is the only one in the cluster/realm/zonegroup.

Reply:
Fixed

Reply:
fixed